code/modules/urist/items/tgitems.dm

```diff
@@ -284,7 +284,7 @@ Please only put items here that don't have a huge definition - Glloyd
 	icon_state = "satchel-flat"
 	w_class = 3 //Can fit in backpacks itself.
 	storage_slots = 5
-	max_combined_w_class = 15
+	max_storage_space = 15 //check values!
```

Comment:
This is more than a var rename, the PR that changed it modified some items' values for that var, so it needs to be tested.